data/com.github.melix99.telegrand.desktop.in.in

```diff
@@ -2,10 +2,12 @@
 Name=Telegrand
 Comment=A Telegram client optimized for the GNOME desktop
 Type=Application
-Exec=telegrand
+TryExec=telegrand
+Exec=telegrand %U
```

Comment:
I don't think this change is needed, IIRC that's useful only if you want to open the app with the link via CLI.

Reply:
Ok, I'll fix it tomorrow. I also have to improve commit messages, cause they're not descriptive enough imo

Reply:
I think it is actually required. Initially I thought the app works without it, but looks like I tested the older version which indeed used these arguments

Reply:
Amberol is using the same arguments and I trust ebassi programming skills